src/actions/index.js

```diff
@@ -0,0 +1,36 @@
+import { FETCH_GITHUB_INFO, FETCH_GITHUB_INFO_SUCCESS, FETCH_GITHUB_INFO_FAILURE, CLEAR_INFO } from './constants'
+import store from '../redux-thunk/store'
+
+export const getGithubInfo = () => ({
+    type: FETCH_GITHUB_INFO
+})
+export const fetchGithubInfoSuccess = data => {
+    return {
+        type: FETCH_GITHUB_INFO_SUCCESS,
+        payload: data,
+    }
+}
+export const fetchGithubInfoFailure = () => {
+    return {
+        type: FETCH_GITHUB_INFO_FAILURE
+    }
+}
+export const fetchGithubInfo = username => {
+    const user = username.replace(/\s/g, "");
+    store.dispatch(getGithubInfo());
+    return (dispatch) => {
+        return fetch(`https://api.github.com/users/${user}`)
```

Comment:
Everything perfect.
**I think** you can pass user url into fetch. 
replace(/\s/g, "") is unnecessary.
You can still keep this code.